homework/Task 2/Lesson.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace homework._Task2
+{
```

Comment:
Fix the architecture, Description has to be a property, public for set. In this case no need to repeat it inhabitant classes